app/src/main/java/com/infomaniak/mail/ui/MainViewModel.kt

```diff
@@ -116,6 +116,14 @@ class MainViewModel : ViewModel() {
         loadContacts()
     }
 
+    fun loadCurrentMailbox() {
+        Log.i(TAG, "loadCurrentMailbox")
+        val mailboxes = loadMailboxes()
```

Comment:
It would be nice to rename this method, its purpose is just to update realm to remote

Reply:
`updateMailboxes`, `updateFolders`, `updateThreads`, etc...
`getCurrentMailbox`, `getCurrentFolder`, etc...